help/policies/code_of_conduct.md

```diff
@@ -0,0 +1,69 @@
+# arXiv Code of Conduct
+
+## Seeking Feedback on Beta Version of Code of Conduct
+  
+Below is the beta version of arXiv's Code of Conduct. We would like to get your feedback! Please let us know what you think [here](https://cornell.qualtrics.com/jfe/form/SV_eQDZib6K66o6gD3). We will be collecting feedback through 18 June 2019.
+
+---
+
+## Opening Statement 
+
+arXiv is a community of authors, readers, moderators, advisory board members, supporting members (libraries, research institutions, etc.), donors, staff, and collaborators in scholarly communication and software development. arXiv is also the  platform where our community engages with one another. We expect all community members to act in accordance with our core values outlined here.
+
+Failure to comply with this Code of Conduct may result in a warning or suspension from participation in arXiv.
+
+### Stewardship
+
+arXiv is an open access resource. We are dedicated to the permanent custodial preservation of the scholarly record and to the continued dissemination of scholarly scientific research. We are sustained by a robust community of scientists from around the world who trust us with their work and ideas, and by the financial support of individuals and organizations. 
+
+arXiv looks to the needs of the international scientific community when we set priorities and while we engage in continual development. We are a partner in ongoing open access development and pleased to share our 30 years of experience.
+
+### Ethics and Responsibility
+
+arXiv community members should uphold intellectual honesty and ethics based on community and academic standards when writing and handling scholarly works. We strive to be objective and transparent in our decisions about policies, standards, and practices.
+
+Community members assume responsibility to represent themselves and their work honestly, to be accountable for their own decisions, and to follow arXiv [policies](#policies). Community members should respect the boundary between personal privacy and scholarly openness.
```

Comment:
Oddly the ``policies](#policies)`` link is not getting rendered correctly (https://marxdown-coc-arxivops-782.development.arxiv.org/help/policies/code_of_conduct) -- the markdown is correct, so will treat as a bug: https://arxiv-org.atlassian.net/browse/ARXIVNG-2178

Reply:
![image](https://user-images.githubusercontent.com/3451594/58574646-71170d80-820e-11e9-8bcc-262f6b2e3f99.png)

Reply:
Maybe just need to rerun the checks? I had a couple edits to get the link right.

Reply:
Oh weird, looks good now